### ISSUE TITLE
[issue]: Ubuntu 22.04.3 not booting with Ventoy

### ISSUE THREAD

**Issue by CherryDT** [ventoy/Ventoy#2631]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.96

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

UEFI Mode

### Partition Style

MBR

### Disk Capacity

64GB

### Disk Manufacturer

SanDisk

### Image file checksum (if applicable)

Yes.

### Image file download link (if applicable)

https://ubuntu.com/download/desktop/thank-you?version=22.04.3&architecture=amd64

### What happened?

When I try to boot Ubuntu 22.04.3, it gets stuck in the state shown in the picture below:

![IMG20231024185625](https://github.com/ventoy/Ventoy/assets/6010463/c07bd86d-4f70-4ac8-8fbd-7c876cce1f68)

This happens both in normal and grub2 mode and both with persistence enabled and disabled.

Ubuntu 20.04 works fine.

The laptop is an ASUS ROG Zephyrus G14 GA401.

**Comment by catherinedoyel** [comment]
Have you tried safe graphics?

**Comment by CherryDT** [comment]
I remember exploring that option but that didn't help. I don't remember whether the option was available but didn't change anything or whether it simply wasn't available at all.

**Comment by catherinedoyel** [comment]
okay, so after you click the Ubuntu iso you will see various options on the first option press e to change the boot parameters. remote `quiet` & `splash`. put in it's place `nomodeset`. This sets it to low resolution mode and will provide much more detail on the screen about what went wrong and when in the brackets is time since the Linux kernel started executing. Without the more verbose logs it's hard to tell if Ventoy, Ubuntu, or Asus issue.

**Comment by DoomTay** [comment]
Could be a machine-specific thing. It boots on a Steam Deck just fine. However, it did NOT boot on my main PC, an ASRock Z170 Extreme3. I don't remember if I tried safe graphics mode.

**Comment by CherryDT** [comment]
Currently traveling, I will try that `nomodeset` when I'm back home.

**Comment by DoomTay** [comment]
I can confirm in my case that booting in safe graphics works fine, so I imagine it'll work in CherryDT's.

**Comment by CherryDT** [comment]
OK that works, but first I was stuck in 800x600. Under "Displays" I didn't see an option to change it.

Then I tried without `nomodeset` (and also still without `quiet splash`), and I got this behavior instead: First, I got stuck for around 2 minutes (not the full 3) here:

![image](https://github.com/ventoy/Ventoy/assets/6010463/19e24cae-7759-4dab-8163-e104eede69d2)

Then it continued a bit further but ultimately got stuck here entirely (cursor also stopped blinking):

![image](https://github.com/ventoy/Ventoy/assets/6010463/7709f895-3cae-47cf-bde0-9f83d2eb6b88)

1) How can I get Ubuntu working with a reasonable screen resolution?
2) Assuming the resolution issue gets resolved, how can I edit my settings in Ventoy in such a way that this works out of the box without having to edit the command line every time?

**Comment by DoomTay** [comment]
Do you see a menu when you first boot to the ISO? There should be an option that says something like "safe graphics".

I think what's happening is that Ubuntu's drivers aren't compatible with whatever GPU your machine has. `nomodeset` is basically telling Ubuntu to boot with very basic graphics drivers. It MIGHT be possible to install proper drivers if you were to install Ubuntu to the machine.

Do you have another spare thumb drive? If so, you could try burning the Ubuntu ISO to it and booting to that. If you're getting the same behavior there, then that probably means that whatever is going on isn't Ventoy's fault

**Comment by CherryDT** [comment]
Yes but safe graphics alone doesn't do the trick, it has `nomodeset` but still has `quiet splash`.

I can try with another flash drive next time.

**Comment by DoomTay** [comment]
Oh, I meant to say burning the ISO by itself using something like Rufus

**Comment by CherryDT** [comment]
Yes, that's what I understood too. Sorry I wasn't clear.
